### PR TITLE
add ssdb cache adapter

### PR DESCRIPTION
**Pull request by lcbluestorm**
add ssdb cache adapter

### CODE REVIEW

**Comment by astaxie** [comment]
@lcbluestorm as you add the dependence `github.com/ssdb/gossdb/ssdb`. you also need to add this `.travis.yml`

**Comment by lcbluestorm** [comment]
I have added github.com/ssdb/gossdb/ssdb to .travis.yml

**Comment by astaxie** [comment]
@lcbluestorm please check the travis check failed

**Comment by lcbluestorm** [comment]
I have built by myself, and there is a problem: maybe need a ssdb server

**Comment by astaxie** [comment]
if it can't integrate with travis, how can we know it works well. How can we setup a ssdb in travis?

**Comment by astaxie** [comment]
@lcbluestorm 

could you help to check with this: https://github.com/eleme/node-ssdb/blob/master/.travis.yml
```
before_install:
  - sudo apt-get update -qq
  - sudo apt-get install -qq libsnappy-dev
  - git clone git://github.com/ideawu/ssdb.git
  - cd ssdb
  - git checkout b62f018872352df9b8fdd69cd3b3a9a9cc8d3e2a
  - make
  - cd ..

before_script:
  - mkdir -p res/var
  - ./ssdb/ssdb-server ./res/ssdb.conf -d

after_script:
  - killall -w ssdb-server
  - rm -rf ./res/var/*
```

**Comment by lcbluestorm** [comment]
ok, I'll check it , thank you

**Comment by lcbluestorm** [comment]
now the ssdb  built succefully:
PASS
ok  	github.com/lcbluestorm/beego/cache/ssdb	0.011s
while the config build failed
FAIL	github.com/lcbluestorm/beego/config/xml [build failed]
FAIL	github.com/lcbluestorm/beego/config/yaml [build failed]

**Comment by astaxie** [comment]
@lcbluestorm looks good now. you can see travis build. https://travis-ci.org/astaxie/beego/builds/114435532

File: .travis.yml [hunk 1]
```diff
@@ -28,10 +34,17 @@ install:
   - go get github.com/siddontang/ledisdb/ledis
   - go get golang.org/x/tools/cmd/vet
   - go get github.com/golang/lint/golint
+  - go get github.com/ssdb/gossdb/ssdb
+  - go get github.com/astaxie/beego/cache
```
Comment (astaxie) [inline .travis.yml]: you don't need to install beego. As it's already in this repo

**Comment by lcbluestorm** [comment]
ok , I have delete 'go get github.com/astaxie/beego/cache' .

**Comment by lcbluestorm** [comment]
ok , I haved deleted 'go get github.com/astaxie/beego/cache'.

File: cache/ssdb/ssdb.go [hunk 2]
```diff
@@ -0,0 +1,240 @@
+package ssdb
+
+import (
+	"encoding/json"
+	"errors"
+	"strconv"
+	"strings"
+
+	"github.com/ssdb/gossdb/ssdb"
+	"time"
```
Comment (astaxie) [inline cache/ssdb/ssdb.go]: could you move time to the top?

**Comment by lcbluestorm** [comment]
ok, I have move time to the top

**Pull request merged**